cmd/lifecycle/creator.go

```diff
@@ -247,6 +268,8 @@ func (c *createCmd) Exec() error {
 		platform:      c.platform,
 		platformDir:   c.platformDir,
 	}.build(group, plan)
+	cancelPing()
+	<-pingDoneChan
```

Comment:
Is there a need to block here? If we don't need the channel, that would make the code a bit simpler.

Reply:
It's not necessary but it shouldn't block for long, I just like to wait for goroutine if possible.

Reply:
It should be pretty instantaneous.